src/admin-views/aggregator/tabs/scheduled.php

```diff
@@ -6,6 +6,9 @@
 <?php echo Tribe__Events__Aggregator__Tabs__Scheduled::instance()->maybe_display_aggregator_missing_license_key_message(); ?>
 <?php $table->views(); ?>
 <form id="posts-filter" method="get">
-<?php $table->nonce(); ?>
-<?php $table->display(); ?>
+<?php
+$table->nonce();
+$table->search_box('Search', 'search_ea_scheduled');
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: ff9601667d987aa6d3cb993dd2f12e70</sub>